builddefs/common_features.mk

```diff
@@ -738,6 +738,12 @@ ifeq ($(strip $(HD44780_ENABLE)), yes)
     SRC += hd44780.c
 endif
 
+ifeq ($(strip $(WS0010_ENABLE)), yes)
```

Comment:
Is there a reason that this is a separate "feature" rather than one of the valid types for `OLED_DRIVER`?

Reply:
I figured it is more of a character display than an oled screen. It's like the old lcd's on calculators and cash registers and stuff. Its basically a replacement for the lcd character display with the hd44780 ic.
Also this is my first PR to an actual project and i'm no programmer so I just followed how the hd44780 driver was implemented. So if you think it should be a type of oled driver then thats fine with me :)

Reply:
oh also its a different ic and communication is 4bit serial instead of i2c like the oled screens